### PR TITLE
New csproj

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: src/NServiceBus.AmazonSQS/Configure/SqsTransport.cs [hunk 1]
```diff
@@ -5,13 +5,25 @@
     using System;
     using Transport;
 
+    /// <summary>
+    /// Sqs transport definition.
```
Comment (SeanFeldman) [inline src/NServiceBus.AmazonSQS/Configure/SqsTransport.cs]: nit: Sqs is an abbreviation. SQS?

**Comment by danielmarbach** [comment]
@Particular/aws-maintainers This already builds on TC but fails when running the acceptance tests. We need to get a few SQS connection string and hook them into TC. Nonetheless, I think we can merge this soon into develop since the build passed.

**Comment by danielmarbach** [comment]
No :( limitation of the new format. You have to copy them locally :( when u
edit the content files it has no influence really. but each repo has
currently this problem.
-- 

Have you tried NServiceBus V6 <https://particular.net/nservicebus-v6> yet?

Learn how to scale applications with microservices and NServiceBus 6
<https://www.pluralsight.com/courses/microservices-nservicebus6-scaling-applications>
.
------------------------------

*Join Udi Dahan's next Advanced Distributed Design Course:*

September 2017 Los Angeles, CA, USA <http://particular.net/adsd>

November 2017 Sydney, Australia <http://particular.net/adsd>
------------------------------

**Comment by ahofman** [comment]
Ok cool, just confirming, good to know. Happy for me to merge @danielmarbach ?

**Comment by danielmarbach** [comment]
Can u remove the WIP first? I did not address @seanfeldman nit comment. Are you ok if we merge nonetheless then yes please go ahead

**Comment by danielmarbach** [comment]
We still have to inspect the build 15 failing tests but I'm self-merging this to unlock other work that has to happen in this repo

**Pull request merged**